src/pages/start-mining/mainnet.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Mine mainnet Stacks tokens
+description: Set up and run a miner on the Stacks 2.0 mainnet
+icon: MainnetIcon
+experience: beginners
+duration: 10 minutes
+tags:
+  - tutorial
+images:
+  large: /images/pages/start-mining.svg
+  sm: /images/pages/start-mining-sm.svg
+---
+
+## Introduction
+
+Make sure you've followed the [Running mainnet node](/understand-stacks/running-mainnet-node) tutorial. Once completed it's only a few more steps to run a proof-of-burn miner on the mainnet.
+
+[@page-reference | inline]
+| /understand-stacks/running-mainnet-node
+
+If you're interested in mining on the testnet, you can find instructions on how to do that here:
+
+[@page-reference | inline]
+| /start-mining/testnet
+
+If you want to learn more about the technical details of mining, please review the mining guide:
+
+[@page-reference | inline]
+| /understand-stacks/mining
+
+## Running bitcoind locally
+
+To participate as a miner on mainnet, you must have access to a mainnet bitcoin node. One way to accomplish this is to run bitcoind locally. [Ensure your computer meets the minimum hardware requirements before continuing.](https://bitcoin.org/en/bitcoin-core/features/requirements)
+
+First, download the bitcoind software for your platform from https://bitcoin.org/en/download.
+
+Next, start bitcoind with the following configuration:
+
+```
+server=1
+rpcuser=your-bitcoind-username
+rpcpassword=your-bitcoind-password
+txindex=0
+listen=1
+rpcserialversion=0
+maxorphantx=1
+banscore=1
+bind=0.0.0.0:8333
+rpcbind=0.0.0.0:8332
+rpcport=8332
+```
+
+Finally, start bitcoind as follows:
+
+```bash
+bitcoind -conf=path/to/bitcoin.conf
+```
+
+It may take a few days for the node to synchronize with the Bitcoin mainnet.
```

Comment:
nit: I'm a bit conflicted with the above indication for the timing being 10 minutes. If the reader for the first time looks at this, it'll take days to complete, do I get this right?

Reply:
Yes that's correct. There may be faster ways we can look into for recommending to users, but it's too close to make a docs change for that now. I'd rather leave it as is, since this is the path we took.